compiler/tests-common/tests/org/jetbrains/kotlin/codegen/AbstractLineNumberTest.java

```diff
@@ -44,11 +44,11 @@
     private static final Pattern TEST_LINE_NUMBER_PATTERN = Pattern.compile("^.*test." + LINE_NUMBER_FUN + "\\(\\).*$");
 
     @NotNull
-    private KotlinCoreEnvironment createEnvironment() {
+    protected KotlinCoreEnvironment createEnvironment(boolean setup) {
         return KotlinCoreEnvironment.createForTests(
                 myTestRootDisposable,
                 KotlinTestUtils.newConfiguration(
-                        ConfigurationKind.JDK_ONLY, TestJdkKind.MOCK_JDK, KotlinTestUtils.getAnnotationsJar(), tmpdir
+                        ConfigurationKind.ALL, TestJdkKind.MOCK_JDK, KotlinTestUtils.getAnnotationsJar(), tmpdir
```

Comment:
Use updateConfiguration/extractConfigurationKind to configure ir stuff (rebase is required)